app/assets/src/components/visualizations/table/BaseTable.jsx

```diff
@@ -96,14 +96,14 @@ class BaseTable extends React.Component {
       <div className={cs.sortableHeader}>
         {columnData ? (
           <ColumnHeaderTooltip
-            trigger={<div className={cs.label}>{label}</div>}
+            trigger={<span className={cs.label}>{label}</span>}
```

Comment:
not sure if this caused it but it looks like the table header titles on the Samples table are no longer showing ellipses and getting smushed together (vs. staging where they do ellipses)

![Screen Shot 2019-09-19 at 5 34 10 PM](https://user-images.githubusercontent.com/5652739/65290370-ce0f3a00-db03-11e9-8e38-e78eee5182bf.png)

Reply:
Not because of this, but should now be fixed.